spec/requests/nonprofits_spec.rb

```diff
@@ -1,53 +1,48 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
 require 'rails_helper'
-require 'support/contexts/shared_donation_charge_context'
 describe NonprofitsController, type: :request do
-  include_context :shared_donation_charge_context
-
+  let(:nonprofit) { create(:nonprofit_base, :with_default_billing_subscription)}
   let(:user) { create(:user_as_nonprofit_associate, nonprofit: nonprofit)  }
-  let(:our_np) do
-    billing_subscription
-    nonprofit
-  end
+
+  # without this, the nonprofit's pages for payments load properly
```

Comment:
Should this comment be `# without this, the nonprofit's pages for payments DOESNT load properly`